benchmarks/transport_montecarlo_packet_trackers.py

```diff
@@ -18,21 +17,17 @@ class BenchmarkTransportMontecarloPacketTrackers(BenchmarkBase):
     repeat = 2
 
     @functools.cache
-    def setup(self, num_packets, length):
+    def setup(self):
         sim = self.simulation_rpacket_tracking_enabled
         self.TransportState = sim.transport.transport_state
 
-    def time_rpacket_trackers_to_dataframe(self, num_packets, length):
+    def time_rpacket_trackers_to_dataframe(self):
         packet_trackers.rpacket_trackers_to_dataframe(
             self.TransportState.rpacket_tracker
         )
 
-    def time_generate_rpacket_tracker_list(self, num_packets, length):
-        packet_trackers.generate_rpacket_tracker_list(num_packets, length)
+    def time_generate_rpacket_tracker_list(self):
```

Comment:
You could parameterize this for different values of num_packets and length.

Reply:
So should I revert back to the last changes since it does parameterize these functions?

Reply:
What I meant was, time_generate_rpacket_last_interaction_tracker_list should not be parameterized for different values of length.

Reply:
Same could be said for `time_rpacket_trackers_to_dataframe`

Reply:
can you suggest some parameter  values that i can use for the whole class?

Reply:
num_packets = (1, 10, 50), length = (1, 10, 50)

Reply:
that might increase the benchmark time since the total number of times each benchmark will run 9 times. Are you sure we want that because you mentioned in the issue about the benchmark action time?

Reply:
What I meant was running generate_rpacket_last_interaction_tracker_list for different values of length was not required and should be removed but we have to run generate_rpacket_tracker_list for different length values.

Reply:
> What I meant was running generate_rpacket_last_interaction_tracker_list for different values of length was not required and should be removed but we have to run generate_rpacket_tracker_list for different length values.

That can be possible but I have to make different classes.

Reply:
Is making class necessary?

Reply:
yes

Reply:
then benchmark it for num_packets = 50, length = 10